presto-docs/src/main/sphinx/functions/bitwise.rst

```diff
@@ -51,6 +51,16 @@ Bitwise Functions
         SELECT bitwise_arithmetic_shift_right(-8, 2); -- -2
         SELECT bitwise_arithmetic_shift_right(7, 2); -- 1
 
+.. function:: bitwise_trailing_zeros(x) -> bigint
+
+    Count the number of trailing zero bits set in ``x`` (treated as
+    ``bits``-bit signed integer) in 2's complement big-endian representation::
```

Comment:
Where is ``bits`` from here?

Reply:
Sorry, that was a copy-paste leftover from `bit_count(x, bits)`. This function uses a Java `long` and bits is always 64.